.changes/unreleased/tinymce-TINY-10854-2024-05-10.yaml

```diff
@@ -0,0 +1,7 @@
+project: tinymce
+kind: Fixed
+body: '`nbsp`s wasn''t replaced correctly if the next character is wrapped in an inline
+  element.'
```

Comment:
Very hard do describe this, my attempt. 😄 

```suggestion
body: 'Extra unnecessary nbsp entities was inserted before and after inline elements created by input of formatted text.'
```

Reply:
ok, changed with yours